tests/unit-tests/test_operation.py

```diff
@@ -109,50 +75,11 @@ def test_add_search_multiple_with_result(self):
         assert operation.op_id == 0
         assert operation.requests == [
             {
-                'details': [
-                    'base="ou=groups,dc=example"',
-                    'scope=2',
-                    'deref=0',
-                    'filter="(cn=group_name)"'
-                ],
-                'verb': 'SRCH'
-            },
-            {
-                'details': [
-                    'attr=memberUid'
-                ],
-                'verb': 'SRCH'
+                'verb': 'SRCH',
+                'details': ['attrs=memberUid', 'base="ou=groups,dc=example"', 'deref=0', 'filter="(cn=group_name)"', 'scope=2'],
             }
         ]
         assert operation.response_verb == "SEARCH RESULT"
         assert operation.response_verb_details == set(
             ['tag=101', 'err=0', 'nentries=1', 'text='])
         assert operation.loggable() == True
-        assert operation.dict() == {'op_id': 0,
```

Comment:
I got ride of the dict spec'ing because it was just getting too annoying to maintain, the prior logic should be more than enough